src/tests/msw-handlers/index.ts

```diff
@@ -152,12 +157,14 @@ type GetDistributionListHandler = ResponseResolver<
 >;
 
 export const registerGetDistributionListHandler = (
-	dl: {
+	data: {
```

Comment:
IMO the DL model can be used as type for this parameters.
Or are the types separated on purpose to isolate the model from the API layer?

Reply:
You convinced me :+1:

Reply:
@gnekoz check on the last commit [8585cce](https://github.com/zextras/carbonio-contacts-ui/pull/187/commits/8585cce0c844dd93ed0e1ab1061426ecd2b32acb)

I changed the handler to denormalize data inside it, and used a builder function to allow the customization of the handler in order to set specific values on the response. I had to allow this, because of the 3-values boolean of the response, against the 2-values boolean of the object. With the tests we are checking all 3 values, so I needed to specifically set them.